resources/functionMap.php

```diff
@@ -2020,7 +2020,7 @@
 'DomXsltStylesheet::result_dump_mem' => ['string', 'xmldoc'=>'DOMDocument'],
 'DOTNET::__construct' => ['void', 'assembly_name'=>'string', 'class_name'=>'string', 'codepage='=>'int'],
 'dotnet_load' => ['int', 'assembly_name'=>'string', 'datatype_name='=>'string', 'codepage='=>'int'],
-'doubleval' => ['float', 'var'=>'mixed'],
+'doubleval' => ['float', 'var'=>'array|bool|float|int|resource|string|null'],
```

Comment:
Is it ever useful to allow the `array` and `resource` here?

Reply:
I'm also a bit confused - `doubleval` does not accept `object` here which should already be picked up by an existing rule (CallToFunctionParametersRule) but apparently isn't. Any idea why?